### PR TITLE
Added unit tests around SOS Aggregation service. Minor refactor/clean up

### PR DESCRIPTION
**Pull request by thongsav-usgs**
to support that testing.

### CODE REVIEW

File: nar-webservices/src/main/java/gov/usgs/cida/nar/service/SiteInformationService.java [hunk 1]
```diff
@@ -62,13 +62,23 @@
...
+	
+	
 	public void streamData(OutputStream output, 
 			final MimeType mimeType,
 			final List<String> siteType,
 			final List<String> stationId,
 			final List<String> state) throws IOException {
 		
-		String wfsUrl = JNDISingleton.getInstance().getProperty(SITE_INFO_URL_JNDI_NAME);
 		String siteLayerName = JNDISingleton.getInstance().getProperty(SITE_LAYER_JNDI_NAME);
```
Comment (jiwalker-usgs) [inline nar-webservices/src/main/java/gov/usgs/cida/nar/service/SiteInformationService.java]: Why not move the layer property to the constructor as well?

File: nar-webservices/test.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jiwalker-usgs) [inline nar-webservices/test.xml]: Is this supposed to be here? Maybe src/test/resources?

**Comment by jiwalker-usgs** [comment]
:+1: just a couple comments

File: nar-webservices/src/main/java/gov/usgs/cida/nar/service/SiteInformationService.java (hunk 1 above)
Comment (thongsav-usgs) [inline nar-webservices/src/main/java/gov/usgs/cida/nar/service/SiteInformationService.java]: :+1: had tunnel vision and didn't see the very next line.

File: nar-webservices/test.xml (hunk 2 above)
Comment (thongsav-usgs) [inline nar-webservices/test.xml]: whoops, was indeed an accidental check in.

**Pull request merged**